### DISCUSSION TITLE
How to modify queries?

### DISCUSSION THREAD

**Discussion by zFlxw** [prisma/prisma#15229] [Q&A] [answered]
Hey all,
first of all, sorry for the weird sounding title, I don't know how to describe my issue better.

I am currently implementing soft delete for user account deactivation. Now I want to run a cron job to delete the accounts that have been deactivated for 30 days or more. I store the timestamp of when the account was disabled in the database. 

Is there any way to add up 30 days to the date of when the account was disabled, so that I can check if the deactivation date + the 30 days period is greater than the current date (and therefore delete the account)?

**Comment by austincrim** [comment, marked as answer]
Hey @zFlxw 👋,

This is definitely something you can do in your app code, possibly in a [transaction](https://www.prisma.io/docs/guides/performance-and-optimization/prisma-client-transactions-guide#interactive-transactions-in-preview). Query the current data, add 30 days, perform the check, and then perform the delete.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
